README.md

```diff
@@ -15,23 +44,28 @@ system to make contributing to this repository easier.
 
 ### Exercises
 
-All exercises have the following structure:
+All exercises are registered in the file
+- `config.json`: Every exercise has a unique name and a difficulty. The sequence order is also the order in which the exercises are to be solved.
+
```

Comment:
This is the kind of explanation that really helps people when they are trying to understand how thing are organized! :+1: